Resources/Prototypes/Catalog/Fills/Crates/botany.yml

```diff
@@ -18,8 +16,6 @@
         amount: 2
       - id: FlyAmanitaSeeds
         amount: 2
-      - id: PineappleSeeds
-        amount: 2
```

Comment:
Pineapple seeds aren't obtainable from anywhere else, you should leave them here, add them to the vendor or add them to the regular seed crate.